virttest/utils_misc.py

```diff
@@ -2338,7 +2338,8 @@ def get_qemu_version(params):
     :return: A dict contain qemu versoins info as {'major': int, 'minor': int,
     'update': int, 'is_rhev': bool}
     """
-    version = {'major': None, 'minor': None, 'update': None, 'is_rhev': False}
+    version = {'major': None, 'minor': None, 'update': None, 'is_rhev': False,
+               'is_ma': False, 'either_rhev_or_ma': False}
```

Comment:
I don't think that either_rhev_or_ma need explicitly parameterized since it can be dertermined by existing parameter:is_rhev,and is_ma

Reply:
Done